src/fswAlgorithms/effectorInterfaces/hingedRigidBodyPIDMotor/hingedRigidBodyPIDMotor.c

```diff
@@ -92,4 +94,14 @@ void Update_hingedRigidBodyPIDMotor(hingedRigidBodyPIDMotorConfig *configData, u
     P = configData->P;
     I = configData->I;
 
+    /*! compute integral term */
+    double dt;
+    if (callTime != 0) {
+        dt = (callTime - configData->priorTime) * NANO2SEC;
+        configData->intError += (thetaError + configData->priorThetaError) * dt / 2;
```

Comment:
`priorThetaError` is never initialized.  Set this to zero in the `Reset()` method.

Reply:
Fixed it.